client/src/app/pages/applications/analysis-wizard/set-targets.tsx

```diff
@@ -27,11 +25,7 @@ export const SetTargets: React.FC = () => {
     refetch: refetchRuleBundles,
   } = useFetchRuleBundles();
 
-  const {
-    bundleOrderSetting,
-    isFetching,
-    refetch: refreshBundleOrderSetting,
-  } = useFetchBundleOrder(ruleBundles);
+  const bundleOrderSetting = useSetting<number[]>("ui.bundle.order");
```

Comment:
I wonder if there is a way to have the `number[]` here automatically inferred by the `"ui.bundle.order"` you're passing in, since we already know the mapping of setting keys to setting value types. Hmm...